src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -232,25 +233,29 @@ public void clickSectionCheckBoxInSectionLevel(int instrNum, int sectionLevelNum
     
     public void clickViewStudentCheckBoxInSectionLevel(int instrNum, int sectionLevelNum) {
         String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " > div > div.panel-body > div.col-sm-6.border-right-gray > input[type=\"checkbox\"]:nth-child(1)";
+                             + " input[type='checkbox'][name='canviewstudentinsectionsectiongroup"
+                             + sectionLevelNum + "']";
         browser.driver.findElement(By.cssSelector(cssSelector)).click();
     }
     
     public void clickViewOthersCommentsCheckBoxInSectionLevel(int instrNum, int sectionLevelNum) {
         String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " > div > div.panel-body > div.col-sm-6.border-right-gray > input[type=\"checkbox\"]:nth-child(5)";
+                             + " input[type='checkbox'][name='canviewcommentinsectionsectiongroup"
+                             + sectionLevelNum + "']";
         browser.driver.findElement(By.cssSelector(cssSelector)).click();
```

Comment:
Suggestion: perhaps can extract a separate element like the `coursePanel` in `clickCourseLevelPrivilegesLink()`

so this will become something like 
~~~
WebElement permissionPanel = this.permissionPanel(instrNum);
String viewPermissionCheckbox = "input[type='checkbox'][name='canviewstudentinsectionsectiongroup"
+ sectionLevelNum + "']";
permissionPanel.findElement(By.cssSelector(viewPermissionCheckbox));
~~~